packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -1,70 +1,308 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { Field, Form, Formik } from 'formik';
+import { Button, Icon, Input, Loading, Table, Popover } from '@deriv/components';
+import { requestWS } from 'Utils/websocket';
 import { localize } from 'Components/i18next';
-import { toSentenceCase } from 'Utils/string';
+import FooterActions from 'Components/footer-actions/footer-actions.jsx';
+import { textValidator } from 'Utils/validations';
+import { generateHexColourFromNickname, getShortNickname } from 'Utils/string';
 import './my-profile.scss';
 
-// TODO: This is just mockup data. it will be removed after Adding BE API.
-const user_data = {
-    first_name: 'johny',
-    last_name: 'bravo',
-    security_deposit: 0,
-    buy_trades: 23,
-    sell_trades: 25,
-    past_month_trades: 42,
-    past_month_completion_rate: 99,
-    average_release_time: 5,
-    average_release_unit: 'min',
-};
+const MyProfile = () => {
+    const [advertiser_info, setAdvertiserInfo] = React.useState({});
+    const [has_poa, setHasPoa] = React.useState(false);
+    const [has_poi, setHasPoi] = React.useState(false);
+    const [is_loading, setIsLoading] = React.useState(true);
+    const [is_mounted, setIsMounted] = React.useState(false);
```

Comment:
Use a `React.useRef` here instead, we don't want to trigger a re-render before unmounting.